src/mpl2/src/mpl.tcl

```diff
@@ -259,11 +259,13 @@ proc rtl_macro_placer { args } {
 sta::define_cmd_args "place_macro" {-macro_name macro_name \
                                     -location location \
                                     [-orientation orientation] \
+                                    [-dont_snap]
```

Comment:
I find negative names to cause more confusion and be harder to discuss in documentation and harder to remember. I would prefer a positive name, like `-exact` as in exact coordinates.